src/config/loadGitHubAppsFromEnvironment.ts

```diff
@@ -53,41 +80,72 @@ export class GitHubAppsRegistry {
   // API for payload (JSON struct coming from GitHub)
 
   loadFromPayload(payload) {
-    // Soon we aim to support multiple orgs!
-    const org = '__tmp_org_placeholder__'; // payload?.organization?.login;
-    if (!org) {
-      throw new Error(`Could not find an org in ${JSON.stringify(payload)}.`);
+    // Org slug is differently accessed in org-scoped APIs vs. repo-scoped APIs
+    let org: string;
+    if (payload.organization) {
+      org = payload.organization.login;
+    } else if (payload.repository?.owner?.type === 'Organization') {
+      org = payload.repository.owner.login;
+    } else {
+      throw new Error(
+        `Could not find an org in '${JSON.stringify(
+          payload.organization
+        )}' or '${JSON.stringify(payload.repository?.owner)}'.`
```

Comment:
Should probs test this.

Reply:
Done in 2a616cd75149940558d9809673111a7886563c54. Writing tests did end up resulting in a couple improvements. 👍